pkgs/applications/gis/qgis/unwrapped.nix

```diff
@@ -156,8 +152,9 @@ in mkDerivation rec {
   '';
 
   cmakeFlags = [
+    "-DCMAKE_BUILD_TYPE=Release"
     "-DWITH_3D=True"
-    "-DWITH_PDAL=TRUE"
+    "-DWITH_PDAL=True"
```

Comment:
```suggestion
    (lib.cmakeBool "WITH_PDAL" true)
```

Reply:
@sikmir , thank you for this suggestion. I already noticed that in other PR. I'll try to implement it in other housekeeping PR for both QGIS packages together.